### PR TITLE
Dmp 1510 v55 v56 schema

### PR DESCRIPTION
**Pull request by chrisdavecm**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DMP-1510

### Change description ###
Changes for v55 schema

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/darts/common/entity/CaseManagementRetentionEntity.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    public static final String ID = "cmr_id";
+    public static final String TABLE_NAME = "case_management_retention";
+
+    @Id
+    @Column(name = ID)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "cmr_gen")
+    @SequenceGenerator(name = "cmr_gen", sequenceName = "cmr_seq", allocationSize = 1)
+    private Integer id;
+
+    @ManyToOne
```
Comment (richardbull) [inline src/main/java/uk/gov/hmcts/darts/common/entity/CaseManagementRetentionEntity.java:30]: Should these relationships be similar to your change in AnnotationEntity?
```
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = ..., nullable = false)
```

File: src/main/java/uk/gov/hmcts/darts/common/entity/CaseRetentionEntity.java [hunk 2]
```diff
@@ -38,13 +38,32 @@ public class CaseRetentionEntity extends CreatedModifiedBaseEntity {
     private CourtCaseEntity courtCase;
 
     @ManyToOne
```
Comment (richardbull) [inline src/main/java/uk/gov/hmcts/darts/common/entity/CaseRetentionEntity.java:40]: Please can you check these ManyToOne relationships be similar to your change in AnnotationEntity?
```
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = ..., nullable = false)
```
Only columns cmr_id, comments, retention_object_id look to nullable in the `retention.sql`

File: src/main/resources/db/migration/common/V1_236__v55_v56_updates.sql [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+ADD CONSTRAINT retention_policy_type_last_modified_by_fk
+FOREIGN KEY (last_modified_by) REFERENCES user_account(usr_id);
+
+ALTER TABLE case_retention
+ADD CONSTRAINT case_retention_case_management_retention_fk
+FOREIGN KEY (cmr_id) REFERENCES case_management_retention(cmr_id);
+
+-- Security
+ALTER TABLE security_group ADD COLUMN use_interpreter  BOOLEAN		NOT null default FALSE;
+ALTER TABLE security_group ALTER COLUMN global_access  SET default FALSE;
```
Comment (richardbull) [inline src/main/resources/db/migration/common/V1_236__v55_v56_updates.sql:103]: Looks like we already have `not null | false` in our schema
```
 global_access | boolean |           | not null | false
 ```

File: src/main/java/uk/gov/hmcts/darts/common/entity/TranscriptionEntity.java [hunk 4]
```diff
@@ -79,6 +79,9 @@ public class TranscriptionEntity extends CreatedModifiedBaseEntity {
     @Column(name = "is_manual", nullable = false)
```
[the remark this answers is not in the document]
Comment (chrisdavecm) [reply src/main/java/uk/gov/hmcts/darts/common/entity/TranscriptionEntity.java:79]: Have done that in a separate PR

**Pull request merged**